### PR TITLE
Initial commit for EnaSolar Integration

### PR DESCRIPTION
**Pull request by geustace**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/63252
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2759
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by geustace** [comment]
This PR seems to have stalled, is there something I need to do before it can be merged?

File: CODEOWNERS [hunk 1]
```diff
@@ -130,6 +130,7 @@ source/_integrations/emby.markdown @mezz64
 source/_integrations/emoncms.markdown @borpin
 source/_integrations/emonitor.markdown @bdraco
 source/_integrations/emulated_kasa.markdown @kbickar
+source/_integrations/enasolar.markdown @geustace
```
Comment (klaasnicolaas) [inline CODEOWNERS:133]: No adjustments are necessary here as it is synced with the codebase.

File: source/_integrations/enasolar.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: EnaSolar Solar Inverter
+description: Instructions on how to connect your EnaSolar GT Solar Inverter to Home Assistant.
+ha_category:
+  - Energy
+ha_iot_class: Local Polling
+ha_release: '0.1.0'
```
Comment (klaasnicolaas) [inline source/_integrations/enasolar.markdown]:
```suggestion
ha_release: 2022.2
```

File: source/_integrations/enasolar.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Note: Polling too frequently caused the web server in the inverter to become unresponsive, to reset turn the AC switch on the inverter Off and On again. The current implementation polls the meters every minute and the daily data every 10 minutes. Polling is also reduced at night.
+
+
+## Configuration
+
+1. Select Integrations from the Home Assistant Configuration page
+2. Select ADD INTEGRATION
+3. Search for EnaSolar.
+
+Fill in the connection form. Note: Naming the inverter isn't particularly helpful unless you have more than one. select NEXT. If the connection succeeded the next form will have been pre-filled with values extracted from the inverter.  You may choose to over-ride a value if is incorrect.
```
Comment (klaasnicolaas) [inline source/_integrations/enasolar.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
When this PR is merged depends on the progress in the codebase 😉

**Comment by geustace** [comment]
Moved documentation into the custom component docs folder.

**Pull request closed without merging**